public/components/event_analytics/explorer/timechart_header/timechart_header.tsx

```diff
@@ -7,19 +7,19 @@ import React from 'react';
 import { EuiFlexGroup, EuiFlexItem, EuiToolTip, EuiText, EuiSelect } from '@elastic/eui';
 import { I18nProvider } from '@osd/i18n/react';
 import { i18n } from '@osd/i18n';
+import moment from 'moment';
+import datemath from '@elastic/datemath';
+import {
+  DATE_DISPLAY_FORMAT,
+  DEFAULT_DATETIME_STRING,
+} from '../../../../../common/constants/explorer';
+
+// TODO: move this function into some common library for other observability datetime uses
```

Comment:
I was thinking about removing this function as a whole, since it can just be done as `.format(DATE_DISPLAY_FORMAT)`.
This function was made back when there was a lot more code involved, but for now I think this is the better option since it doesn't seem like this function will get much other use. What do you think?

Reply:
@paulstn Yes, do that change.